runtime/integration-tests/src/pools/utils/env.rs

```diff
@@ -704,13 +711,17 @@ fn test_env(
 		);
 		let client = Arc::new(client);
 		let clone_client = client.clone();
-		let instance = INSTANCE_COUNTER.fetch_add(1, sp_std::sync::atomic::Ordering::SeqCst);
-		assert!(FudgeInherentTimestamp::new(
-			instance,
+
+		let instance_id = FudgeInherentTimestamp::create_instance(
+			std::time::Duration::from_secs(6),
+			Some(std::time::Duration::from_millis(START_DATE)),
+		);
+		assert!(FudgeInherentTimestamp::force_new(
+			instance_id,
 			std::time::Duration::from_secs(6),
 			Some(std::time::Duration::from_millis(START_DATE)),
 		)
-		.is_none());
+		.is_ok());
```

Comment:
Remove. Instance is already created above.

Reply:
Done.